src/App.js

```diff
@@ -51,6 +52,10 @@ const App = () => {
                 homepageUrlHref="https://gov.uk"
             />
             <Navigation/>
+            <Switch>
+                <Route path="/about" exact render={()=>(<BackToTop mode="overlay"/>)} />
+                <Route path="/" exact render={()=>(<BackToTop mode="overlay"/>)} />
+            </Switch>
```

Comment:
@MikeCoats Could you add a brief comment on this so devs won't get confused in the future?